modules/javafx.controls/src/test/java/test/javafx/scene/control/SpinnerTest.java

```diff
@@ -337,7 +338,7 @@ public void expectExceptionWhenOneArgsDecrementCalled_noValueFactory() {
      *                                                                         *
      **************************************************************************/
 
-    @Ignore("Need KeyboardEventFirer")
+    @Ignore("JDK-8328701")
```

Comment:
the spinner is not editable and no commitValue() is issued (please see my comment in the ticket)

https://bugs.openjdk.org/browse/JDK-8328701?focusedId=14659352&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-14659352

Reply:
In that case, maybe you can repurpose that JBS bug ID into a test bug? Either way, using that bug ID to skip this test seems good.